### ISSUE TITLE
Feature Request: Add possibility to split configuration into several files

### ISSUE THREAD

**Issue by lucas-dclrcq** [oknozor/toml-bombadil#22]
bombadil.toml can get quite big over time. It would help to be able to better organize the configuration by allowing the user to split the file into smaller ones.

The user would  add one or several import configurations in the main conf file :

```
# Simple import
[[import]]
source = "another.conf.toml"

# Folder import (any regex should work)
[[import]]
source = "config.d/*.toml"
```

The external configuration file(s) should simply be inject in the main configuration file.

**Comment by oknozor** [comment]
#28 merged in 2.0.0, still needs to be documented.

**Comment by oknozor** [comment]
documented in 7581fa6 I am closing this for now and opening a new issue to support regex in import paths.
